astropy/io/votable/table.py

```diff
@@ -49,12 +53,17 @@ def parse(source, columns=None, invalid='exception', pedantic=None,
             - 'mask': mask out invalid values
 
     pedantic : bool, optional
-        When `True`, raise an error when the file violates the spec,
-        otherwise issue a warning.  Warnings may be controlled using
-        the standard Python mechanisms.  See the `warnings`
-        module in the Python standard library for more information.
-        When not provided, uses the configuration setting
-        ``astropy.io.votable.pedantic``, which defaults to False.
+        This is now an alias for ``verify='warn'`` (if ``pedantic`` is `False`)
```

Comment:
Not sure if `pedantic` should be documented in docstring at all, according to precedence in #5171 .

Reply:
Fixed